doc/source/index.rst

```diff
@@ -31,22 +31,24 @@ such as PostGIS.
    Documentation <docs>
    Community <community>
 
-Get in touch
+.. container:: button
+
+    :doc:`Getting started <getting_started>` :doc:`Documentation <docs>`
+    :doc:`About GeoPandas <about>` :doc:`Community <community>`
+
+Useful links
 ------------
 
-- Ask usage questions ("How do I?") on `StackOverflow`_ or `GIS StackExchange`_.
-- Report bugs, suggest features or view the source code `on GitHub`_.
-- For a quick question about a bug report or feature request, or Pull Request,
-  head over to the `gitter channel`_.
-- For less well defined questions or ideas, or to announce other projects of
-  interest to GeoPandas users, ... use the `mailing list`_.
```

Comment:
Could we keep this list somewhere? (either here, or maybe on the "community" landing page?

Reply:
It is in Getting started. This was a duplication.